source/includes/v2/_assets.md

```diff
@@ -6,10 +6,15 @@ The asset is a generic model which represents a native asset, fungible token or
 It's possible that in the different blockchains assets with the same symbol and/or address are existed, so you have to rely only on the asset ID if you need to deterministically identify it.
 If you've received a deposit of an asset which is not supported by Sirius yet, you can add this particular asset to your personal list by specifying all needed asset parameters.
 
+**Rest API:** `POST /api/v2/assets`
+
+**gRPC API:** `swisschain.sirius.api.v2.assets`
+
```

Comment:
Is this related to entire assets v2 service/controller? If so, why `POST` then?

Reply:
`+`